main.go

```diff
@@ -44,6 +45,7 @@ const (
 	apikeyid = "<API-KEY-ID>"
 	apikey   = "<API-KEY>"
 	partID   = "<PART-ID>"
+	webhook  = "<WEBHOOK>"
```

Comment:
[q] since there was a big push to move all of the public repos out of viamrobotics, do we want to go ahead and make this repo private & put the secrets here?

Reply:
hmm I didn't know there was a big push for that... yeah I think that's reasonable

Reply:
going to do that in a follow up PR though if that's alright with you

Reply:
yeah i thought thats what the whole viam-modules thing was about, and def doesn't happen in this pr

Reply:
oh I thought that was just for viam supported modules, not a script like this, but I still think we can make it private and include the secret. it's less complicated that way with making sure the secrets are there and also not accidentally committing them